compiler/plugins/input/StableHLO/Conversion/test/auto_input_conversion.mlir

```diff
@@ -8,3 +8,13 @@ func.func @simple_add_stablehlo(%arg0: tensor<2x2xi32>, %arg1: tensor<2x2xi32>)
   %0 = stablehlo.add %arg0, %arg1 : (tensor<2x2xi32>, tensor<2x2xi32>) -> tensor<2x2xi32>
   return %0 : tensor<2x2xi32>
 }
+
+// -----
+
+// CHECK-LABEL: util.func public @vhlo_func
+vhlo.func_v1 @vhlo_func(%arg0: !vhlo.tensor_v1<!vhlo.i32_v1>) -> (!vhlo.tensor_v1<!vhlo.i32_v1>) {
+  // CHECK: arith.constant
+  %0 = "vhlo.constant_v1"() <{value = #vhlo.tensor_v1<dense<1> : tensor<i32>>}> : () -> !vhlo.tensor_v1<!vhlo.i32_v1>
```

Comment:
Could test a few versions of VHLO to ensure the stablehlo deserialization works.

Maybe find an interesting op from the tests in https://github.com/openxla/stablehlo/tree/main/stablehlo/tests/vhlo and verify that old and new versions of that op (e.g. versions that have different attributes / text formats) both deserialize into the same StableHLO that the rest of the input conversion pipeline here can handle.

Reply:
I added 3 extra tests (one comparison, one dot and one gather) in [a226b0d](https://github.com/iree-org/iree/pull/16999/commits/a226b0de411d69ee5133aa44b3207336a8158ac4)